src/app.js

```diff
@@ -18,7 +19,27 @@ library.add(faSignInAlt, faSignOutAlt, faCloudDownloadAlt, faCloudUploadAlt);
 
 Vue.component('font-awesome-icon', FontAwesomeIcon);
 
-Vue.use(Vuetify);
+Vue.use(Vuetify, {
+  theme: {
```

Comment:
I think it might be better to keep the theme general, so like `backgroundDark` rather than singling out specific use cases here, like `search`. This way if you're reading the search code you can see and change the color there without having to come here, and the very high level section here isn't polluted with use-case-specific color settings